src/warehouse/warehouse.ts

```diff
@@ -0,0 +1,35 @@
+import { env } from 'decentraland-commons'
+
+const WAREHOUSE_URL: string = env.get('WAREHOUSE_URL')
+const WAREHOUSE_TOKEN: string = env.get('WAREHOUSE_TOKEN')
+const WAREHOUSE_CONTEXT_PREFIX: string = env.get('WAREHOUSE_CONTEXT_PREFIX')
+
+/**
+ * Sends data to the warehouse to be stored.
+ *
+ * @param context - The name of the application storing data.
+ * @param event - The name of the event that we are registering.
+ * @param body - The contet of the event that we are registering.
+ */
+export async function sendDataToWarehouse(
+  context: string,
+  event: string,
+  body: any
+): Promise<void> {
+  const data = {
+    context: WAREHOUSE_CONTEXT_PREFIX
+      ? `${WAREHOUSE_CONTEXT_PREFIX}-${context}`
+      : context,
+    event,
+    body,
+  }
+
+  await fetch(WAREHOUSE_URL, {
```

Comment:
I think this is working here because the project is importing `isomorphic-fetch` like this: `import 'isomorphic-fetch'` on `graphClient.ts`. Worst still, the project also has a `node-fetch` dependency 🤦 

We don't need to fix this here of course, but maybe we can at least add a `import fetchfrom 'node-fetch'` at the top of the file, to be responsible